website/siteConfig.js

```diff
@@ -84,6 +84,12 @@ const siteConfig = {
   // template. For example, if you need your repo's URL...
   repoUrl,
   usePrism: ['jsx'],
+  
+  // search
+  algolia: {
+    apiKey: 'cd9b8b73f97b64ed04570e41c507683f',
```

Comment:
where this key comes from?

Reply:
This is randomly created during the application https://community.algolia.com/docsearch/apply.html

Feel free to provide me your email address so that I grant you access to the analytics we provide for free.

Reply:
Thanks! Just applied for a key, it may be interesting to see the analytics :)

Reply:
Great, invited!